packages/ag-charts-community/src/chart/chart.ts

```diff
@@ -352,7 +355,11 @@ export abstract class Chart extends Observable {
             ctx.regionManager.listenAll('click', (event) => this.onClick(event)),
             ctx.regionManager.listenAll('dblclick', (event) => this.onDoubleClick(event)),
             seriesRegion.addListener('hover', (event) => this.onMouseMove(event)),
-            seriesRegion.addListener('leave', (event) => this.onLeave(event)),
+            horizontalAxesRegion.addListener('hover', (event) => this.onMouseMove(event)),
+            verticalAxesRegion.addListener('hover', (event) => this.onMouseMove(event)),
+            seriesRegion.addListener('leave', (event) => this.onLeave(event), All),
```

Comment:
There's a risk that the `All` parameter might interrupt animations when the mouse-pointer leaves this bounding box. I would advice checking for a regression using these repro steps:

1. Load any example with animation. The [Marker vs. Path Animation](https://charts-staging.ag-grid.com/javascript/line-series-test/#marker-vs-path-animation) example is a good test case because the animation is very slow there.
2. Move the mouse pointer over the series rect.
3. Move the mouse pointer out of the chart.

Reply:
Tested, seems like no regression but removed it anyway